lib/services/network/http_service.dart

```diff
@@ -33,6 +243,7 @@ class HttpService extends GetxService {
   /// Global try-catch function
   Future<Response> runApiGuarded(Future<Response> Function() func, {bool checkOrigin = true}) async {
     if (http.origin.isEmpty && checkOrigin) {
+      Logger.info("Api failed ${StackTrace.current}");
```

Comment:
If you want to keep this can you make it an `error` log

Reply:
This was originally designed to catch http attempted calls when developing rustpush backend, I think it's a good idea tho.

Reply:
This was originally designed to catch http attempted calls when developing rustpush backend, I think it's a good idea tho.